### DISCUSSION TITLE
windows iso editing

### DISCUSSION THREAD

**Discussion by eskehacettepe** [ventoy/Ventoy#2732] [General]
How can I integrate a driver and program into the codes here? I need an inf or exe for the driver so that it can be installed automatically after the bios.
https://resimlink.com/rShPNb-gy

**Comment by steve6375** [comment]
https://forums.ventoy.net/
Use the Forum for general questions. GitHub Issues are for Issues with the Ventoy code, not how to use Ventoy.

You can also try reading the Ventoy documentation. https://ventoy.net/en/plugin_injection.html
VentoyAutoRun.bat (1.0.55+)
Since Ventoy 1.0.55, if you inject a X:\VentoyAutoRun.bat file, Ventoy will run this bat file automatically.
Please refer [About VentoyAutoRun.bat](https://ventoy.net/en/doc_inject_autorun.html) for details.
